taskana-adapter-taskana-connector/src/main/java/pro/taskana/adapter/taskanaconnector/config/TaskanaSystemConnectorConfiguration.java

```diff
@@ -61,14 +61,15 @@ public ClassificationService getClassificationService(TaskanaEngine taskanaEngin
 
   @Bean
   @Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
-  public TaskanaEngine taskanaEngine(TaskanaEngineConfiguration taskanaEngineConfiguration) {
+  public TaskanaEngine taskanaEngine(TaskanaEngineConfiguration taskanaEngineConfiguration)
+      throws SQLException {
     return taskanaEngineConfiguration.buildTaskanaEngine();
   }
 
   @Bean
   @Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
   public SpringTaskanaEngineConfiguration taskanaEngineConfiguration(
-      @Qualifier("taskanaDataSource") DataSource taskanaDataSource) throws SQLException {
+      @Qualifier("taskanaDataSource") DataSource taskanaDataSource) {
```

Comment:
This is already done on master branch. Probably a merge conflict?

Reply:
Yea probably